humanoid_league_game_controller/scripts/sim_gamestate.py

```diff
@@ -48,6 +64,8 @@ def get_key():
     gameState = GameStateMsg()
     gameState.header.stamp = rospy.Time.now()
 
+    # init secondary state team to our teamID
+    gameState.secondaryStateTeam = 8  # this is the default TeamID in the behavior
```

Comment:
We are usually setting the team id via the ros parameter `/team_id`. I think it would be better to use the parameter if it is present and fall back to `8` when it is unset.

Reply:
The team_id parameter is load on the parameter server by the game controller package, I think. So the team_id parameter will never be on the parameter server when using this script. But it can be changed anyway.